### ISSUE TITLE
Add ability to upload patient photo

### ISSUE THREAD

**Issue by esmithayer** [codeforamerica/rva-screening#115]
Users wanted a place to add a photo of the patient to make sure they have the right person--some patients use multiple names, or occasionally there are multiple patients with the same name and DOB.

**Comment by mapsam** [comment]
Below is an example of a patient image, with key details added as a new column. The form is therefore pushed into a new column of slightly smaller width to make those big 100% width inputs a bit less insane.

![rva-screener-patientphoto](https://cloud.githubusercontent.com/assets/1943001/7925270/9bcdc4e8-0877-11e5-8d90-f205d0673356.gif)

**Comment by mapsam** [comment]
apparently LICEcap isn't the best with motion screens ... but hopefully that conveys the point :stuck_out_tongue:

**Comment by bengolder** [comment]
:+1:

**Comment by mapsam** [comment]
Also important to note the image stays between tabs ... what are your thoughts on that?

![rva-screener-patientphoto2](https://cloud.githubusercontent.com/assets/1943001/7925327/2fe8adfa-0878-11e5-8217-1704dc4a5db7.gif)

**Comment by bengolder** [comment]
I think it's fine for now. Whether or not it should be displayed in the print version seems like the biggest question. I do think it adds some valuable context while moving between actions, so a user might be less likely to forget whose profile they're viewing the edit history for or who they're about to send to a service.

**Comment by mapsam** [comment]
Agreed! I think the print view is likely to change significantly with #145 potentially replacing this need.
